PlayCover/Views/App Views/PlayAppView.swift

```diff
@@ -207,6 +207,13 @@ struct PlayAppConditionalView: View {
                     Text(app.name)
                         .foregroundColor(selected?.info.bundleIdentifier == app.info.bundleIdentifier ?
                                          selectedTextColor : Color.primary)
+
```

Comment:
The style of this exclamation point doesn't match up with the badge in IPA Library. In IPA Library, the icon is unfilled, whereas here, it is filled. In IPA Library, the icon is on the left, here, it's on the right. In IPA Library, it's included in the selection, here, it is not.

<img width="1012" alt="Screenshot 2022-10-19 at 22 39 30" src="https://user-images.githubusercontent.com/42140194/196843403-fc347b25-fd16-445b-9c0c-87531ae862c1.png">
<img width="1012" alt="Screenshot 2022-10-19 at 22 39 36" src="https://user-images.githubusercontent.com/42140194/196843417-e58c6237-bd52-4dcd-b57b-2f91ef28f016.png">